convert2rhel/checks.py

```diff
@@ -448,3 +457,81 @@ def _bad_kernel_substring(kernel_release):
         )
         return True
     return False
+
+
+def check_package_updates():
+    """Ensure that the system packages installed are up-to-date."""
+    logger.task("Prepare: Checking if the packages are up-to-date")
+
+    # Get a total count of how many packages is left to update
+    total_packages = get_total_packages_to_update()
+
+    if total_packages > 0:
+        logger.warning(
+            "The system has %s packages not updated. "
+            "If you wish to update then before continuing with the conversion, execute the following step:\n\n"
+            "Run: yum update -y\n" % total_packages
+        )
+    else:
+        logger.info("System is up-to-date.")
+
+
+def is_loaded_kernel_latest():
+    """Check if the loaded kernel is behind or of the same version as in yum repos."""
+    logger.task("Prepare: Checking if the loaded kernel version is the most recent.")
+
+    # For OL/CentOS 8 the `kernel` is just a meta package, instead, we check for `kernel-core`.
+    # But for 6 and 7 releases, the correct way to check is using `kernel`.
+    package_to_check = "kernel-core" if system_info.version.major >= 8 else "kernel"
```

Comment:
It means that the tool will be working in standard RHEL environment with default provided kernels.